packages/xo-web/src/xo-app/sr/tab-xostor.js

```diff
@@ -0,0 +1,122 @@
+import _ from 'intl'
+import ActionButton from 'action-button'
+import Component from 'base-component'
+import Icon from 'icon'
+import React from 'react'
+import SortedTable from 'sorted-table'
+import PifsColumn from 'sorted-table/pifs-column'
+import { addSubscriptions, connectStore } from 'utils'
+import { Card, CardBlock, CardHeader } from 'card'
+import { Col, Container, Row } from 'grid'
+import { createGetObjectsOfType, createSelector } from 'selectors'
+import { find } from 'lodash'
+import { generateId } from 'reaclette-utils'
+import { createXostorInterface, destroyXostorInterface, setXostor, subscribeXostorInterfaces } from 'xo'
+
+const INTERFACES_COLUMNS = [
+  {
+    name: _('name'),
+    itemRenderer: iface => iface.name,
+    sortCriteria: iface => iface.name,
+  },
+  {
+    name: _('pifs'),
+    itemRenderer: (iface, { pifIdsByIfaceName }) => <PifsColumn pifs={pifIdsByIfaceName[iface.name]} />,
+  },
+]
+
+@connectStore({
+  pifs: createGetObjectsOfType('PIF').filter((_, props) => pif => pif.$pool === props.sr.$pool),
+})
+@addSubscriptions(({ sr }) => ({
+  interfaces: subscribeXostorInterfaces(sr),
+}))
+export default class TabXostor extends Component {
+  _actions = [
+    {
+      handler: ifaces => Promise.all(ifaces.map(iface => destroyXostorInterface(this.props.sr, iface.name))),
```

Comment:
Should we have a bulk `destroyXostorInterfaces` so that we don't force refresh the subscriptions too many times in a row?